aws/rhel-8.5-aarch64/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "user": "ec2-user",
+    "runnerArch": "aarch64",
+    "prepareScript": "sudo tee /etc/yum.repos.d/redhat.repo <<EOT\n[RHEL-8-NIGHTLY-BaseOS]\nname=baseos\nbaseurl=http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.5.0/compose/BaseOS/aarch64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-8-NIGHTLY-AppStream]\nname=appstream\nbaseurl=http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.5.0/compose/AppStream/aarch64/os/\nenabled=1\ngpgcheck=0\n\n[RHEL-8-NIGHTLY-CRB]\nname=crb\nbaseurl=http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.5.0/compose/CRB/aarch64/os/\nenabled=1\ngpgcheck=0\nEOT"
```

Comment:
Please rebase to `main` and rename the repository file to `rhel8internal.repo`. See the last commit for details why.

Reply:
Updated.